lib/plugins/entry-files-manifest.js

```diff
@@ -9,26 +9,40 @@
 
 'use strict';
 
-const EntryFilesManifestPlugin = require('../webpack/entry-files-manifest-plugin');
 const PluginPriorities = require('./plugin-priorities');
-const path = require('path');
 const sharedEntryTmpName = require('../utils/sharedEntryTmpName');
-const manifestKeyPrefixHelper = require('../utils/manifest-key-prefix-helper');
+const AssetsPlugin = require('assets-webpack-plugin');
+
+function processOutput(assets) {
+    if (assets.hasOwnProperty(sharedEntryTmpName)) {
+        delete assets[sharedEntryTmpName];
+    }
+    for (const asset in assets) {
```

Comment:
Can we add a little comment about what this is doing / why?

Reply:
true I never thought of that. Yes we could turn it into an array. I will add that to this PR.

Reply:
@weaverryan Is there anything else we need to do to the output?